python/pyspark/ml/feature.py

```diff
@@ -2436,6 +2458,26 @@ def from_labels(cls, labels, inputCol, outputCol=None, handleInvalid=None):
             model.setHandleInvalid(handleInvalid)
         return model
 
+    @classmethod
+    @since("3.0.0")
+    def from_ArrayOfLabels(cls, arrayOfLabels, inputCols, outputCols=None,
```

Comment:
I'm not sure about our Python naming conventions, but would `from_array_of_labels` be more conventional? or does this follow another similar method?

Reply:
+1 for `from_array_of_labels`.